### ISSUE TITLE
Create models for all tables in MySQL

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/starchart#97]
Similar to what's happening in #96, we need to create files in `app/models/*` for all of our data models:

- user
- record
- certificate
- etc.

These should hide the details of working with Prisma, similar to how the user model works https://github.com/Seneca-CDOT/starchart/blob/c86bbe336d8ca9a75a5157bb326b9827c9bd7756/app/models/user.server.ts

The specific API to use needs some thought.  We'll also need some tests.

**Comment by cychu42** [comment]
I thought this issue is just for making functions back then.
I'm not sure I'll get to tests this week, so I'm dissociating this from milestone 0.3 for now.

**Comment by humphd** [comment]
I think we should close this now, and continue work in other issues.  @cychu42 can you do that, unless you disagree for some reason?

**Comment by cychu42** [comment]
Yes, we can close this to mark CRUD as "done". Testing is mentioned in the other issue mentioned.
